### PR TITLE
updated changes for market aggregator script

### PR DESCRIPTION
**Pull request by karthikyadav3**

### CODE REVIEW

File: d3a_api_client/setups/test_market_aggregator.py [hunk 1]
```diff
@@ -58,22 +66,16 @@ def on_finish(self, finish_info):
     "domain_name": domain_name,
     "websockets_domain_name": websockets_domain_name
 }
-
+market_names = ["house 1"]
 house_1_uuid = get_area_uuid_from_area_name_and_collaboration_id(
-    market_args["simulation_id"], "House 1", market_args["domain_name"])
+    market_args["simulation_id"], market_names[0], market_args["domain_name"])
```
Comment (hannesdiedrich) [inline d3a_api_client/setups/test_market_aggregator.py:71]: Is this change for later, when we want to connect to more than one market?

File: d3a_api_client/setups/test_market_aggregator.py [hunk 2]
```diff
@@ -5,6 +5,10 @@
     get_sim_id_and_domain_names
 from d3a_api_client.rest_market import RestMarketClient
 
+# import logging
+logger = logging.getLogger()
+logger.disabled = False
```
Comment (hannesdiedrich) [inline d3a_api_client/setups/test_market_aggregator.py:10]: Why are you disabling the logger?

File: d3a_api_client/setups/test_market_aggregator.py [hunk 3]
```diff
@@ -21,16 +25,20 @@ def on_market_cycle(self, market_info):
         logging.info(f"AGGREGATOR_MARKET_INFO: {market_info}")
         if self.is_finished is True:
             return
-        if "content" not in market_info:
-            return
-        self.fee_cents_per_kwh += 1
+        # if "content" not in market_info:
+        # return
```
Comment (hannesdiedrich) [inline d3a_api_client/setups/test_market_aggregator.py:29]: Please remove all commented out code. Thank!

**Review by hannesdiedrich — COMMENTED** [review]
Left some comments. Also, please create a new package under `d3a_api_client/setups` called `e2e-tests` and put this code into a new module. thanks!

File: d3a_api_client/setups/test_market_aggregator.py (hunk 1 above)
Comment (karthikyadav3) [reply d3a_api_client/setups/test_market_aggregator.py:71]: Yes, If we add more houses then we have to add the house name to the list and access its index value again.

File: d3a_api_client/setups/test_market_aggregator.py (hunk 2 above)
Comment (karthikyadav3) [reply d3a_api_client/setups/test_market_aggregator.py:10]: I added it in the first line. I will delete the commented lines in the code.

File: d3a_api_client/setups/test_market_aggregator.py (hunk 3 above)
Comment (karthikyadav3) [reply d3a_api_client/setups/test_market_aggregator.py:29]: Sure

**Pull request closed without merging**